gadget/device/encrypt.go

```diff
@@ -108,9 +108,10 @@ var ErrNoSealedKeys = errors.New("no sealed keys")
 type SealingMethod string
 
 const (
-	SealingMethodLegacyTPM    = SealingMethod("")
-	SealingMethodTPM          = SealingMethod("tpm")
-	SealingMethodFDESetupHook = SealingMethod("fde-setup-hook")
+	SealingMethodLegacyTPM      = SealingMethod("")
+	SealingMethodTPM            = SealingMethod("tpm")
+	SealingMethodFDESetupHook   = SealingMethod("fde-setup-hook")
+	SealingMethodNextGeneration = SealingMethod("next-generation")
```

Comment:
Once next-generation is old and is surpased by deep-tpm-nine, will we know what is the meaning of next-generation? Perhaps this should have a comment or a more descriptive name?

Reply:
This is a placeholder.  I am sure we will find a name. Though we could make the last version, make everything newer use this value but have a different way to identify itself. So technically, it could be "FinalFrontier". I am fine with any name. As long as it is not from the alternate timeline.